Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -203,12 +203,10 @@ class Country
 
 		std::set<std::string>& editAllies() { return allies; }
 
-		[[nodiscard]] std::map<std::string, int> getGPInfluences() const { return GPInfluences; }
 		void addGPInfluence(std::string GPTag, int influenceValue) { GPInfluences.insert({GPTag, influenceValue}); }
 		[[nodiscard]] const std::map<std::string, double>& getSpherelings() const { return spherelings; }
-		void addSphereling(std::string);
-		double getSpherelingAutonomy(std::string) const;
-		void setSpherelingAutonomy(std::string spherelingTag, double autonomy);
+		void addSphereling(std::string sphereling) { spherelings.insert(make_pair(sphereling, 0.5)); }
+		void setSpherelingAutonomy(std::string sphereling, double autonomy) { spherelings.find(sphereling)->second = autonomy; }
```

Comment:
Wait, why'd the check that a sphereling was actually found go away? This now has the potential to cause a crash if sphereling isn't in spherelings.

Reply:
Because it's GP->getSpherelings->setSpherelingAutonomy and
`getSpherelings() const { return spherelings; }`

So, no chance of there not being a sphereling

Reply:
Ah, cool then.